src/build/prepareElectronApp.ts

```diff
@@ -78,10 +82,18 @@ async function maybeCopyScripts(srcs: string[], dest: string): Promise<void> {
   }
 
   log.debug(`Copying ${srcs.length} files to inject in app.`);
-  for (const src of srcs) {
+
+  for (let src of srcs) {
+    if (isGistRawUrl(src)) {
+      const fileDetails = await downloadTmpFile(src);
+      log.debug(`Downloaded gist file to ${fileDetails}`);
+
+      src = fileDetails;
+    }
+
     if (!fs.existsSync(src)) {
       throw new Error(
-        `File ${src} not found. Note that Nativefier expects *local* files, not URLs.`,
+        `File ${src} not found. Note that Nativefier expects *local* files, or raw gist URLs.`,
```

Comment:
I expect people to be confused about what a "raw" gist URL is. Please improve error handling to precise what that means.

(And maybe this point is moot if you go by cybermoloch's suggestion to not limit to Gists URLs)